src/controller/index.ts

```diff
@@ -345,6 +346,42 @@ export default class SimulariumController {
         });
     }
 
+    private hackApplyHardcodedAOSettings(fileName: string): void {
```

Comment:
as it turns out, when I tried the same settings on our other trajectories, the change was not making things look worse.  This is subjective so I would like to get lots of eyes on this.

Reply:
I tried these out, I know the falloff and threshold are distance related but I think having the aoradius2 go back down to 1.2 when you get really close gets rid of the added darkness

Reply:
Overall though, looks good to me on all the models. so you're going to get rid of the specific blood plasma settings?

Reply:
Because these settings seem ok generally, then yes I could take out the special case test.   

On the other hand I'm not optimistic about it lasting forever.  I've been thinking more about just letting these settings be embedded in trajectory files as json so each trajectory can have a chance to have its own settings.  Just not sure what the procedure would be to get the settings in there in the first place.